nixos/doc/manual/release-notes/rl-2105.xml

```diff
@@ -804,6 +804,16 @@ environment.systemPackages = [
      the deprecated <option>services.radicale.config</option> is used.
     </para>
    </listitem>
+   <listitem>
+    <para>
+     In the <option>security.acme</option> module, use of <literal>--reuse-key</literal>
+     parameter  for Lego has been removed. It was introduced for HKPK, but this security
```

Comment:
```suggestion
     parameter for Lego has been removed. It was introduced for HKPK, but this security
```

Reply:
My mistake, as this is merged please submit a change to fix and backport this if you are interested. Personally, this is too minor for my taste.